MatrixSDK/MXSession.h

```diff
@@ -59,6 +59,11 @@ typedef enum : NSUInteger
      Data from the MXStore has been loaded.
      */
     MXSessionStateStoreDataReady,
+    
+    /**
+     Data from local cache is being processed.
+     */
+    MXSessionStateProcessingLocalCache,
```

Comment:
Maybe `MXSessionStateProcessingBackgroundSyncCache` for the same reason.
It is not generic at all but it indicates what it does.

Reply:
Updated.